enterprise/server/remote_execution/execution_server/execution_server.go

```diff
@@ -275,6 +276,9 @@ func (s *ExecutionServer) Dispatch(ctx context.Context, req *repb.ExecuteRequest
 	if err != nil {
 		return "", err
 	}
+
+	tracing.AddStringAttributeToCurrentSpan(ctx, "task_id", executionID)
```

Comment:
What do you think about renaming the `ExecutionID` field in `ExecutionTask` to simply `ID` and always referring to it as task ID in code? We have inconsistent references to this ID both as "execution ID" and "task ID".

Reply:
The discrepency was intentional because the task queuing system originated after the remote execution system and for a while had different IDs. At some point i made the IDs the same because it seemed the right thing to do for tracing etc.

Anyway, this is fine if you want to do it -- but they do refer to slightly different things (primary ID of a task vs primary ID of an execution + ident returned to bazel)